rpcclient/examples/customcommand/README.md

```diff
@@ -0,0 +1,32 @@
+Namecoin Core Custom Command Example
+====================================
+
+This example shows how to use custom commands with the rpcclient package, by
+implementing the `name_show` command from Namecoin Core.
+
+## Running the Example
+
+The first step is to use `go get` to download and install the rpcclient package:
+
+```bash
+$ go get github.com/btcsuite/btcd/rpcclient
+```
+
+Next, modify the `main.go` source to specify the correct RPC username and
+password for the RPC server of your Namecoin Core node:
+
+```Go
+	User: "yourrpcuser",
+	Pass: "yourrpcpass",
+```
+
+Finally, navigate to the example's directory and run it with:
+
+```bash
+$ cd $GOPATH/src/github.com/btcsuite/btcd/rpcclient/examples/customcommand
+$ go run *.go
+```
```

Comment:
This example assumes that the user is using Go modules, which is no longer the default with newer Go versions. I'd suggest replacing 9-13 with:

The first step is to clone the `btcd` repo:

```bash
$ git clone github.com/btcsuite/btcd
$ cd btcd
```

And then replace 23-28 with:

Finally, run the example:

```bash
$ go run rpcclient/examples/customcommand/main.go
```

Reply:
This Readme is copied nearly verbatim from the `bitcoincorehttp` example's Readme; for consistency I prefer not to make the requested changes as part of this PR (but I'd be fine with those changes as a separate PR that covers both examples' Readmes).

Reply:
I agree with the changes suggested here, but I'm happy to take care of it in a separate PR.